mycli/main.py

```diff
@@ -811,16 +811,29 @@ def format_output(self, title, cur, headers, expanded=False,
             output.append(title)
 
         if cur:
-            rows = list(cur)
+            column_types = None
+            if hasattr(cur, 'description'):
+                def sanitize(col):
+                    if col is converters.through:
```

Comment:
It looks like PyMySQL has other converters that we'd need to check for.

I think you can just do `return col if type(col) is type else text_type`